slidez-fe-core/src/common/components/interactive-elements/reactions/Reaction.tsx

```diff
@@ -0,0 +1,75 @@
+import { Button, ButtonGroup, Popover } from '@material-ui/core'
+import React, { useState } from 'react'
+import like from '../../../../assets/svgs/like_reaction.svg'
+import thumb_up from '../../../../assets/svgs/thumb_up_reaction.svg'
+import 'animate.css'
+import './reaction.scss'
+
+interface ButtonProps extends React.ButtonHTMLAttributes<HTMLButtonElement> {
+    anchor: HTMLButtonElement | null
+}
+
+const Reaction = ({ anchor }: ButtonProps) => {
+    const [showLike, setShowLike] = useState(false)
+    const [showThumbUp, setShowThumbUp] = useState(false)
+    const [anchorEl, setAnchorEl] = useState<HTMLButtonElement | null>(anchor)
+
+    const handleLike = () => {
+        setShowLike(true)
+        setTimeout(() => setShowLike(false), 2000)
+    }
+
+    const handleThumbUp = () => {
+        setShowThumbUp(true)
+        setTimeout(() => setShowThumbUp(false), 2000)
+    }
+
+    const handleClose = () => {
+        setAnchorEl(null)
+    }
+    const open = Boolean(anchorEl)
+
+    return (
+        <div className='reaction'>
+            <Popover
+                id='simple-popover'
+                anchorEl={anchorEl}
+                onClose={handleClose}
+                open={open}
+                anchorOrigin={{
+                    vertical: 'top',
+                    horizontal: 'left',
+                }}
+                transformOrigin={{
+                    vertical: 'bottom',
+                    horizontal: 'left',
+                }}
+            >
+                <ButtonGroup>
+                    <Button onClick={handleLike}>Like</Button>
+                    <Button onClick={handleThumbUp}>Thumb up</Button>
+                </ButtonGroup>
+            </Popover>
+            {showLike ? (
+                <img
+                    src={like}
+                    alt='like'
+                    className='animate__animated animate__slideInUp'
+                />
+            ) : (
+                ''
+            )}
+            {showThumbUp ? (
+                <img
+                    src={thumb_up}
+                    alt='thumb up'
+                    className='animate__animated animate__slideInUp'
+                />
+            ) : (
+                ''
+            )}
```

Comment:
Make it a function which will get show: boolean and src parameter and return this structure